services/web/server/src/simcore_service_webserver/clusters/models.py

```diff
@@ -22,6 +22,32 @@ class Config:
 CLUSTER_NO_RIGHTS = ClusterAccessRights(read=False, write=False, delete=False)
 
 
+class Authentication(BaseModel):
```

Comment:
TIP: if ``Authentication`` is a base (i.e. cannot be used alone and must be subclassed) you should show that intention by e.g. changing the name it as ``BaseAuthentication`` instead (analogous to ``BaseModel``) or making it abstract?